components/_lists.scss

```diff
@@ -0,0 +1,22 @@
+.wds-list {
+  display: block;
+  list-style-type: none;
+  padding: 0;
+  margin: 18px;
+
+  .wds-small-list-item {
+    display: flex;
+    flex-direction: column;
+    font-size: $typescale-size-minus-1;
+    height: 36px;
```

Comment:
if an item will have more content to fit into one line
this will break

instead of height we could use flex with min height